prime-router/settings/organizations-local.yml

```diff
@@ -365,6 +368,62 @@
           initialTime: 00:00
           timeZone: EASTERN
 
+  - name: gu-doh
+    description: Guam Department of Health
```

Comment:
GU and VT are in organizations-local but not in organizations-prod.  By design, I presume.